nva-datamodel-java/src/main/java/no/unit/nva/model/Publication.java

```diff
@@ -390,6 +394,14 @@ public boolean satisfiesFindableDoiRequirements() {
         return FindableDoiRequirementsValidator.meetsFindableDoiRequirements(this);
     }
 
+    public Set<URI> getCuratingInstitutions() {
+        return nonNull(curatingInstitutions) ? curatingInstitutions : Collections.emptySet();
```

Comment:
Why not just return null? There is a difference "not set" and "no curating institutions"

Reply:
@sondrev For the sake of the receiver, in serialization